src/views/shared/PasswordRequirements.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/FactorUtil',
+],
+function (Okta, FactorUtil) {
+  return Okta.View.extend({
+    template: `{{#if requirements}}
+      <div class="password-requirements--header">Password requirements: </div>
```

Comment:
Should `Password requirements:` be hardcoded like this?

Reply:
Will move to i18n